amivapi/studydocs/endpoints.py

```diff
@@ -80,6 +81,11 @@
                 'maxlength': 30,
                 'nullable': True,
                 'type': 'string'
+            },
+            'coursesemester': {
+                    'maxlength': 5,
+                    'nullable': True,
+                    'type': 'string'
```

Comment:
Ah, i can see your point, but then we need to add more validation, see my other comment